xbmc/interfaces/legacy/InfoTagMusic.h

```diff
@@ -67,6 +67,24 @@ namespace XBMCAddon
 #ifdef DOXYGEN_SHOULD_USE_THIS
       ///
       /// \ingroup python_InfoTagMusic
+      /// @brief \python_func{ getDbId() }
+      ///-----------------------------------------------------------------------
+      /// Get identification number of tag in database.
+      ///
+      /// @return [integer] database id
+      ///
+      ///
+      ///-----------------------------------------------------------------------
+      ///
```

Comment:
I think this returns -1 in case there is no dbid? might be worth adding a comment for that.